### PR TITLE
Fix the Wrapf -> Errorf transition in the pkg

### PR DESCRIPTION
**Pull request by vagababov**
The cleanup for the go.1.13

/assign @adrcunha

### CODE REVIEW

File: test/spoof/spoof.go [hunk 1]
```diff
@@ -241,7 +240,7 @@ func (sc *SpoofingClient) Poll(req *http.Request, inState ResponseChecker) (*Res
 	}
 
 	if err != nil {
-		return resp, errors.Wrapf(err, "response: %s did not pass checks", resp)
+		return resp, fmt.Errorf("response: %s did not pass checks: %w", err)
```
Comment (chizhg) [inline test/spoof/spoof.go]:
```suggestion
		return resp, fmt.Errorf("response: %s did not pass checks: %w", resp, err)
```

**Comment by adrcunha** [comment]
```profiling/server.go:86:17: Errorf format %w has unknown verb w```

knative/pkg Prow jobs are not using Go 1.13:

```
>> go version
go version go1.12 linux/amd64
```

@chaodaiG is it safe to migrate pkg yet?

**Comment by chaodaiG** [comment]
In my opinion pkg should be the last one to migrate, since many other repos depending on pkg haven't migrated to go1.13 yet

**Comment by adrcunha** [comment]
> In my opinion pkg should be the last one to migrate, since many other repos depending on pkg haven't migrated to go1.13 yet

Agree 100%. I like the cleanup though. Victor, can we just rollback the 1.13-specific changes?

**Comment by vagababov** [comment]
I am fine with holding this up, until the others migrate as well.
/hold
/assign @chaodaiG 
Pease unhold when the others are also migrated.

**Comment by evankanderson** [comment]
+1 to holding this until clients have had a chance to move to Go 1.13.

**Comment by chaodaiG** [comment]
> I am fine with holding this up, until the others migrate as well.
> /hold
> /assign @chaodaiG
> Pease unhold when the others are also migrated.

will do

**Comment by vagababov** [comment]
/close
this is way outdated

**Pull request closed without merging**